src/data/faq.json

```diff
@@ -439,6 +439,14 @@
                             "<a href='#vac_cert_booster' target='_self' rel='noopener noreferrer'>To the index of the booster vaccination</a>"
                         ]
                     },
+                    {
+                        "title": "What is booster vaccination and how does it appear in the CWA?",
+                        "anchor": "vac_booster_basics",
+                        "active": true,
+                        "textblock": [
+                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://www.coronawarn.app/assets/screenshots/2-14/en/android/PersonDetailsFragment_cwa_2.png' target='_blank' rel='noopener'><img src='../../assets/screenshots/2-14/en/android/PersonDetailsFragment_cwa_2.png' alt='image' width='300'></a></div><div class='col-sm'><p>Booster vaccination (revaccination) is a vaccination process designed to refresh waning vaccination protection after basic immunization. In most cases, but not always, it takes place according to the recommendations of the authorities after a period of about 5-6 months after the last vaccination procedure. (Example: completion of the vaccination series for basic immunization on 08.07.2021, possible booster vaccination on: 08.01.2022)</p><p>Each booster vaccination can be attested by a digital COVID certificate, which you can scan into the Corona-Warn-App. Important: please ensure that the personal data (surname, first name and date of birth) are shown to match the existing vaccination certificates. Only then will it be possible to assign the new certificate to the booster vaccination of the basic immunization. (Example: same academic title or indication of the full double name).</p><p>Booster shots provide immediate vaccine protection.</p><p>Based on the previous vaccination series, the booster certificate will be documented '3 of 3' (in case of previous basic vaccination with 2 doses) or with '2 of 2' (as vaccination optimization according to the Janssen agent from Johnson & Johnson). In the latter case, however, the CovPassCheck app, Corona-Warn-App and CovPass app cannot display this correctly in all cases for technical reasons. See also: <a href='https://www.coronawarn.app/en/faq/#vac_booster_jj' target='blank' rel='noopener noreferrer'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p><p>The '1 of 1' label for a booster vaccination indicates that it is the first and only vaccination after a COVID-19 disease recovery. Vaccinations with Johnson & Johnson are an exception. See also: <a href='https://www.coronawarn.app/en/faq/#vac_booster_jj' target='blank' rel='noopener noreferrer'>Why does a 14-day wait for full vaccine protection after a booster vaccination appear in the Corona-Warn-App?</a></p></div></div></div>"

```

Comment:
Change text to
"A booster vaccination (revaccination) ..."

Change hyperlink to
`<a href='/assets/screenshots/2-14/en/android/PersonDetailsFragment_cwa_2.png' target='_blank'><img src='/assets/screenshots/2-14/en/android/PersonDetailsFragment_cwa_2.png' alt='image' width='300'></a>`

Reply:
Suggest also changing alt='image' to alt="Certificate List"

Reply:
For dates, write in English not German format:

"Example: completion of the vaccination series for basic immunization on 08 Jul 2021, possible booster vaccination on: 08 Jan 2022"

Using letters instead of numbers for the month is better because then it fits English (United Kingdom) and English (United States).